### ISSUE TITLE
Is this ready to use?

### ISSUE THREAD

**Issue by tinpont** [hden/socketio.objc#1]
seems not completed yet.

**Comment by hden** [comment]
As you can see, it's still work in progress. PR's are totally welcomed. :)

**Comment by hellpf** [comment]
I'm looking forward to this library will be ready to use! It looks very nice! When do you plan to release it?

**Comment by hden** [comment]
@tinpont @hellpf As a matter of fact, I have released a working version right [here](https://github.com/hden/SocketIO.JSCore)!

The main difference are:
- https://github.com/hden/SocketIO.JSCore use JavaScriptCore to bridge to the official client.
- https://github.com/hden/socketio.objc use pure obj-c implementation and is currently under development.

**Comment by hellpf** [comment]
Thank you very much for such a quick reply! I'll try for now the JSCore version.

**Comment by tinpont** [comment]
Thanks for sharing.

**Comment by jmny** [comment]
Hi @hden, could you give us  an update ? What are your plans about this client ?

**Comment by hden** [comment]
@jmny Currently we are missing several key feature e.g. `utf-8 support`, `binary payload`

TL;DR it's not production ready.
